app/api/events.py

```diff
@@ -63,6 +63,15 @@ class Meta:
                            schema='SponsorSchema',
                            many=True,
                            type_='sponsor')
+    notification = Relationship(
+        attribute='notification',
+        self_view='v1.event_notification',
+        self_view_kwargs={'id': '<id>'},
+        related_view='v1.notification_detail',
```

Comment:
Please check the above comment. I think the event notification relations can be added once other APIs are also created. In the model, event has relationship for notification email. So this can be added in a subsequent PR.